openerp/addons/base/res/res_partner.py

```diff
@@ -87,6 +70,9 @@ def fields_view_get_address(self, cr, uid, arch, context={}):
                     tree = etree.fromstring(v)
                     # needs to apply modifiers
                     for childnode in tree.getchildren():
+                        if 'use_parent_address' in self._all_columns:
```

Comment:
I don't think so we are here on children of `adress_format` css class:
``doc.xpath("//div[@class='address_format']")``